repoman_docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gentoo/stage3-amd64
+
+
+RUN emerge-webrsync && emerge --sync && emerge app-crypt/gnupg repoman dev-vcs/git
+
+ENV FEATURES sign
+ENV PORTAGE_GPG_DIR ~/.gnupg/
+
+RUN mkdir -p /usr/local/portage
+RUN printf '\nPORTDIR_OVERLAY="/usr/local/portage"' >> /etc/portage/make.conf
```

Comment:
I am just setting this up so that repoman is launched via a Docker container, so gentoo ebuilds can be generated on systems like Ubuntu.

Is there a point to configuring `repos.conf`?